test/test.js

```diff
@@ -0,0 +1,28 @@
+const puppeteer = require('puppeteer');
+const pageURl = 'http://localhost:3003';
+
+
+let page;
+let browser;
+const width = 1280;
+const height = 720;
+
+beforeAll(async () => {
+  browser = await puppeteer.launch({
+    headless: false,
+    slowMo: 80,
+    args: [`--window-size=${width},${height}`]
+  });
+  page = await browser.newPage();
+  await page.setViewport({ width, height });
+});
+
+afterAll(() => {
+  browser.close();
+});
+
+describee('search function' () => {
+
```

Comment:
I'm guessing tests are to come later? 
also please  fix spelling on 'describee'

Reply:
got it. thanks!